### ISSUE TITLE
GUI "Starting tabulation..." message should appear as soon as button is clicked

### ISSUE THREAD

**Issue by HEdingfield** [BrightSpots/rcv#59]
Fix "Starting tabulation..." message not appearing until after tabulation is complete (even though timestamp is accurate once it's finally displayed).

**Comment by CalebKleppner** [comment]
This still seems to be the case on 8/9/18.  Will relate to logging info to console, GUI and audit files.

**Comment by HEdingfield** [comment]
This is most likely a thread concurrency issue. I've tried to address it so far by putting each of these at the start of `Main.executeTabulation()`:
```
      final CountDownLatch latch = new CountDownLatch(1);
      new Thread(() -> {
        Logger.allLogs(Level.INFO, "Starting tabulation...");
        latch.countDown();
      }).start();
      try {
        latch.await();
      } catch (InterruptedException e) {
        e.printStackTrace();
      }
```

And also:
```
      Logger.allLogs(Level.INFO, "Starting tabulation...");
      try {
        Thread.sleep(1000);
      } catch (InterruptedException e) {
        e.printStackTrace();
      }
```

But neither has worked. It's actually running in real-time in the console though, which implies it's the updating of the GUI TextArea that's not getting executed until later.

I checked  `Logger.addGuiLogging` and @moldover added the following line which could definitely explain this behavior:
`Platform.runLater(() -> textArea.appendText(formatter.format(record)));`

However, even when I change it to:
`textArea.appendText(formatter.format(record));`

It still doesn't appear when it should. I also tried inserting `Thread.sleep` at the start of `Tabulator.tabulate()` and no dice.

Finally, I attempted to go the nuclear option, declaring `Main.executeTabulation()`, `Main.parseCastVoteRecords()`, and `Tabulator.tabulate()` as `synchronized`, but even that didn't work.

@tarheel or @moldover: any ideas?
